frontend/pages/ManageControlsPage/Scripts/components/ScriptDetailsModal/ScriptDetailsModal.tsx

```diff
@@ -71,8 +78,9 @@ const ScriptDetailsModal = ({
   } = useQuery<any, Error>(
     ["scriptContent", selectedScriptDetails?.script_id],
     () =>
-      selectedScriptDetails
-        ? scriptAPI.downloadScript(selectedScriptDetails.script_id!)
+      selectedScriptDetails?.script_id
+        ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I like this. Should we bring to frontend sync to suggest removing this rule? Tired of commenting `//
this will actually always be defined`

Reply:
yeah, Gabe and I are merp about it too, lets def chat at sync,